.github/workflows/build.yml

```diff
@@ -1,30 +1,27 @@
-name: Build documentation pages
-
+name: Build Examples
 on:
   push:
-    branches:
-    - '*'
-#  pull_request_target:
-#    branches:
-#    - '*'
-
+    branches-ignore:
```

Comment:
I get why you ignored `main` here, but there were several occasions where we broke the `main` in the past. Are we 100% secure not to break `main` again with this new config flow? One more thing is that our Github Actions Status Badge in `README.md` is working on `main`, so I am not sure it will continue working after merging this change.

Reply:
We have nightly builds now, so we should continue to ignore main on pushes.